src/callbacks.py

```diff
@@ -16,20 +16,17 @@
 # Otherwise the rf-country-dropdown is not yet defined before we switch tabs
 
 
-@callback(
-    Output('rf-country-dropdown', 'value'),
-    Input('memory-output', 'data')
-)
+@callback(Output("rf-country-dropdown", "value"), Input("memory-output", "data"))
```

Comment:
Just curious, are you using some autoformat on save?

Reply:
We probably should have some linter we all agree to use on save, so that formatting changes like this don't show up in the diff  (Makes it hard to find where actual code changes were made)

Reply:
https://github.com/UBC-MDS/DSCI-532_2024_1_TBtracker/issues/77